doc/source/development/contributing_documentation.rst

```diff
@@ -157,21 +157,3 @@ just built::
 
 And you'll have the satisfaction of seeing your new and improved documentation!
 
-.. _contributing.dev_docs:
-
-Building main branch documentation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-When pull requests are merged into the Xorbits ``main`` branch, the main parts of
```

Comment:
Removing `readthedocs` from CI doesn't stop building the docs of main branch. `readthedocs` will still build the docs whenever a pr got merged to main.

You can keep the content before `see also the CI section.`